### ISSUE TITLE
ColecoVision has no reset button

### ISSUE THREAD

**Issue by meshuggahtas** [TASVideos/BizHawk#992]
Reset button where I would use: Tomarc the Barbarian starts the game in 3 possible place (one of the bottom 3 rooms). According to the manual, the * (star) key should start the game, but it's also started instantly and I can't do anything besides this.

Yes, this is a 2 bug in 1 issue.

**Comment by alyosha-tas** [comment]
.........................ColecoVision OTL

Well I guess I've procrastinated fixing this core for long enough. The Turbo controller never did work right and the whole core is really old and outdated. I guess it's finally time to bring it up to par with the rest of BizHawk.

**Comment by alyosha-tas** [comment]
http://www.atarihq.com/danb/files/colecovision.pdf

As far as I can tell from these schematics, reset just triggers the reset pins on various chips in the console. From the emulator's point of view, this is equivalent to a hard reset (I'm pretty sure anyway.)

The game does appear to auto-start itself, as it does so in every video I can find of it. I don't know why the manual indicates that the game doesn't start automatically in this way.

The '*' key can restart the game after a game over though, which can change where you start.

So, I'm not sure there are actually any bugs here to fix.

**Comment by meshuggahtas** [comment]
Thank you for investigating the issue and sorry for my initial assumptions being wrong (as I refered to the manual). 

Either way, let me write down my informations I collected so far regarding this, **note that these might be invalids**
- There is a ROM that includes both Tomarc and Motocross in one cartridge. It's probably made (only?) for Coleco (maybe for a2600??)
- My initial assumption came also from the fact that many sites includes their screenshot as Tomarc sitting in the middle of the screen (as he would when first spawned in a cave) starting in either at the very left side (bottom row 1st cave) or the middle (bottom row 2nd cave) while BizHawk currently starts at bottom row 3rd cave.
